packages/document/main-doc/docs/en/guides/advanced-features/rspack-start.mdx

```diff
@@ -82,9 +82,7 @@ For example, if you are using pnpm, you can update the Rspack version with `over
 {
   "pnpm": {
     "overrides": {
-      "@rspack/core": "nightly",
-      "@rspack/plugin-react-refresh": "nightly",
-      "@rspack/plugin-html": "nightly"
+      "@rspack/core": "nightly"
```

Comment:
maybe we still need `@rspack/plugin-react-refresh`?

Reply:
updated!